pkgs/build-support/rust/fetchcargo.nix

```diff
@@ -42,7 +43,12 @@ stdenv.mkDerivation {
     ${cargoUpdateHook}
 
     mkdir -p $out
-    cargo vendor $out | cargo-vendor-normalise > config
+    ${if flagNoMergeSources
+      then ''
+        cargo vendor --no-merge-sources $out > config
```

Comment:
Is this incompatible with our normalize script?

Reply:
Currently it is incompatible. Is there any reason to believe the formatting details will change, other than that they don't guarantee it won't? Did we actually observe this? It shouldn't be a problem to adapt the script though.

Reply:
We talked to the author of cargo-vendor and there is no guarantee that the output will stay the same after updating `cargo-vendor`. By having the script we can make sure it will or we can create workarounds.